### PR TITLE
docs: update jwt acl modules

### PR DESCRIPTION
**Pull request by zmstone**
For entperprise edition, the plugins are hidden.
only exposing the modules (with UI) to users.

--- this is why the copy-pates of the text.

### CODE REVIEW

File: en_US/modules/jwt_authentication.md [hunk 1]
```diff
@@ -46,3 +46,59 @@ eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJkYXRhIjp7ImF1dGhvciI6IndpdndpdiIsInNpdGU
 The above JWT Token is only for testing and can be generated with related tools according to your own business needs. An online generation tool is provided here: https://www.jsonwebtoken.io/.
 
 :::
+
+## ACL information stored in claims
+
+The 'ACL Claim Name' filed can be used to to specify which JWT token claim is to be used for ACL rules.
```
Comment (ieQu1) [inline en_US/modules/jwt_authentication.md]:
```suggestion
The 'ACL Claim Name' field can be used to to specify which JWT token claim is to be used for ACL rules.
```
?

File: en_US/advanced/acl-jwt.md [hunk 2]
```diff
@@ -94,5 +92,5 @@ To make ACL rules valid forever, a client may not provide `exp` claim at all.
 
 ::: warning
 1. Using long-living JWTs is not considered secure.
-2. Turning on ACL caching may invalidate the expiration time check.
+2. When ACL cache is enabled, the ACL rule's expiration is either when the cache or JWT expires, whichever comes the last.
```
Comment (thalesmg) [inline en_US/advanced/acl-jwt.md]:
```suggestion
2. When ACL cache is enabled, the ACL rule's expiration is either when the cache or JWT expires, whichever is the later.
```
?

File: en_US/modules/jwt_authentication.md (hunk 1 above)
Comment (savonarola) [inline en_US/modules/jwt_authentication.md]:
```suggestion
The 'ACL Claim Name' field can be used to to specify which JWT token claim is to be used for ACL rules.
```

**Review by ieQu1 — DISMISSED** [review]

File: en_US/modules/jwt_authentication.md [hunk 3]
```diff
@@ -46,3 +46,59 @@ eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJkYXRhIjp7ImF1dGhvciI6IndpdndpdiIsInNpdGU
...
+## ACL expiration
+
+JWT ACL engine will prohibit all operations after the deadline specified in `exp` JWT claim, so
+a client with an expired JWT has to reconnect with a fresh JWT.
+
+To make ACL rules valid forever, a client may not provide `exp` claim at all.
+
+::: warning
+1. Using long-living JWTs is not considered secure.
+2. When ACL cache is enabled, the ACL rule's expiration is either when the cache or JWT expires, whichever comes the last.
```
Comment (thalesmg) [inline en_US/modules/jwt_authentication.md]:
```suggestion
2. When ACL cache is enabled, the ACL rule's expiration is either when the cache or JWT expires, whichever is the later.
```
?

File: en_US/modules/jwt_authentication.md [hunk 4]
```diff
@@ -46,3 +46,59 @@ eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJkYXRhIjp7ImF1dGhvciI6IndpdndpdiIsInNpdGU
...
+    }
+}
+```
+
+## ACL expiration
+
+JWT ACL engine will prohibit all operations after the deadline specified in `exp` JWT claim, so
+a client with an expired JWT has to reconnect with a fresh JWT.
+
+To make ACL rules valid forever, a client may not provide `exp` claim at all.
```
Comment (thalesmg) [inline en_US/modules/jwt_authentication.md]:
```suggestion
To make ACL rules valid forever, a client may omit the `exp` claim.
```

**Review by savonarola — DISMISSED** [review]

**Review by thalesmg — DISMISSED** [review]

**Pull request merged**